pkg/pull/pull.go

```diff
@@ -158,6 +163,7 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 	}
 
 	log.ActionWithSpinner("Pulling upstream")
+	io.WriteString(pullOptions.ReportWriter, "Pulling upstream\n")
```

Comment:
why line breaks?

Reply:
if i dont write the newline you will never see the message